frontend/src/components/nodes/ValidatorTelemetryRow.tsx

```diff
@@ -61,11 +67,20 @@ const ValidatorTelemetryRow: FC<Props> = ({
                   {producedBlocks !== undefined &&
                   expectedBlocks !== undefined ? (
                     <>
-                      {((producedBlocks / expectedBlocks) * 100).toFixed(3)}%
-                      &nbsp;
-                      <span>
-                        ({producedBlocks}/{expectedBlocks})
-                      </span>
+                      <OverlayTrigger
+                        placement={"bottom"}
+                        overlay={
+                          <Tooltip id="produced-blocks-chunks">
+                            {`produced ${producedBlocks}/${expectedBlocks} blocks,`}{" "}
+                            {producedChunks}/{expectedChunks}&nbsp;chunks
```

Comment:
@frol Am I right that `producedChunks` and `expectedChunks` depends of `producedBlocks` & `expectedBlocks`?

Reply:
Yes, that is correct